authui/src/copy.ts

```diff
@@ -17,9 +17,14 @@ function copyToClipboard(str: string): void {
 
 export class CopyButtonController extends Controller {
   static targets = ["source", "button"];
+  static values = {
+    timeoutHandle: Number,
+  };
 
   declare sourceTarget: HTMLElement;
   declare buttonTarget: HTMLButtonElement;
+  declare timeoutHandleValue: number | undefined;
+  declare hasTimeoutHandleValue: boolean;
```

Comment:
In this particular case, we can store the handle in an ordinary instance variable instead of a Stimulus value.